README.md

```diff
@@ -2,38 +2,285 @@
+     },
+     "x509CertificateProperties": {
+        "ekus": [
+           "1.3.6.1.5.5.7.3.3"
+       ],
+       "keyUsage": [
+         "digitalSignature"
+       ],
+       "subject": "CN=Test Signer",
+       "validityInMonths": 12
+     }
+   }
+   EOF
+
+   # create self-signed certificate
+   az keyvault certificate create -n $certName --vault-name $keyVault -p @selfSignedPolicy.json
+
+   # get the key identifier
+   keyID=$(az keyvault certificate show -n $certName --vault-name $keyVault --query 'kid' -o tsv)
+   ```
+4. Prepare a container registry. You can follow the [guide](https://learn.microsoft.com/azure/container-registry/container-registry-get-started-portal?tabs=azure-cli) to create an Azure Container Registry which is recommended. Suppose the logging server is `notation.azurecr.io` with an `hello-world:v1` image.
+5. Sign the container image with Notation:
+   ```sh
+   notation sign notation.azurecr.io/hello-world:v1 --id $keyID --plugin azure-kv
+   ```
+
+   example output
+   ```
+   Warning: Always sign the artifact using digest(@sha256:...) rather than a tag(:v1) because tags are mutable and a tag reference can point to a different artifact than the one signed.
+   Successfully signed notation.azurecr.io/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
+   ```
+6. Verify the signature associated with the image:
+   ```sh
+   # add self-signed certificate to notation trust store
+   cat <<EOF > ./selfSignedCert.crt
+   -----BEGIN CERTIFICATE-----
+   $(az keyvault certificate show -n $certName --vault-name $keyVault --query 'cer' -o tsv)
+   -----END CERTIFICATE-----
+   EOF
+   notation cert add --type ca --store selfSigned ./selfSignedCert.crt
+   
+   # add notation trust policy
+   notationConfigDir="${HOME}/.config/notation"                        # for Linux
+   # notationConfigDir="${HOME}/Library/Application Support/notation"  # for macOS
+
+   mkdir -p $notationConfigDir
+   cat <<EOF > $notationConfigDir/trustpolicy.json
+   {
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            "name": "trust-policy-example",
+            "registryScopes": [ "*" ],
+            "signatureVerification": {
+                "level" : "strict" 
+            },
+            "trustStores": [ "ca:selfSigned" ],
+            "trustedIdentities": [
+                "*"
+            ]
+        }
+    ]
+   }
+   EOF
+   chmod 600 $notationConfigDir/trustpolicy.json
+   ```
+
+   verify signature
+   ```
+   notation verify notation.azurecr.io/hello-world:v1
+   ```
+   example output
+   ```
+   Warning: Always verify the artifact using digest(@sha256:...) rather than a tag(:v1) because resolved digest may not point to the same signed artifact, as tags are mutable.
+   Successfully verified signature for notation.azurecr.io/hello-world@sha256:f54a58bc1aac5ea1a25d796ae155dc228b3f0e11d046ae276b39c4bf2f13d8c4
+   ```
+
+## Getting started with a certificate signed by a trusted CA
```

Comment:
```suggestion
## Sign and verify an artifact with a certificate signed by trusted CA in Azure Key Vault
```

Reply:
This section has a lot of overlap with the previous section. Instead of re-hashing the steps from the previous section, call out the specific differences, such as the policy change, using the csr, and creation of leaf certificate.

Reply:
Updated the doc structure to split getting started 2 sections into 2 files.